src/components/contact-page/GetInTouch.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+
+import { faEnvelope, faLocationDot, faPhone } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { SubmitHandler, useForm } from "react-hook-form";
+
+import { getFieldClassName } from "../general/services/getFieldClassName";
+import alternateCircle from "../../assets/images/alternate_circle.png";
+import { useNotifications } from "../../hooks/useNotifications";
+import { usePostGeneralInquiry } from "./hooks/usePostGeneralInquiry";
+import { GetInTouchFormValues } from "./models/GetInTouchFormShapes";
+import "./css/get-in-touch.css";
+
+export const GetInTouch = () => {
+  const { addNotification } = useNotifications();
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm<GetInTouchFormValues>({
+    defaultValues: {
+      name: "",
+      email: "",
+      message: "",
+    },
+  });
+
+  const { mutate: postGeneralInquiry, error: backendErrors } = usePostGeneralInquiry();
+
+  const onSubmit: SubmitHandler<GetInTouchFormValues> = (formValues) => {
+    postGeneralInquiry(formValues, {
+      onSuccess: () => {
+        addNotification("Thanks for contacting us!", {
+          appearance: "success",
+        });
+      },
+      onError: (err: Error) => {
+        addNotification("There was a problem, please try again later.", {
+          appearance: "error",
+        });
+      },
+    });
+  };
+
+  const blockClassName = "get-in-touch";
+  const errorClassPrefix = `${blockClassName}__input`;
+
+  return (
+    <section className="get-in-touch">
+      <div className="get-in-touch__contents">
+        <img
+          alt=""
+          className="get-in-touch__alternate-circle"
+          role="presentation"
+          src={alternateCircle}
+        />
+        <h2 className="get-in-touch__section-heading">Get in touch</h2>
+        <p className="get-in-touch__text_center"></p>
```

Comment:
should we just remove this and add padding or margin as needed to `h2`?

Reply:
I left it as a placeholder, because the wireframe had `text here`. I can either delete the element, leave it empty, or put something in. `Get in touch: we want to hear from you!`?